scss/base/_variables.scss

```diff
@@ -0,0 +1,8 @@
+/**
+ * @file
+ * File contains variables used for the app (e.g. path variables and color variables etc.)
+ */
+  
+// Material icons related variables
+$material-font-name: 'Material Icons';
```

Comment:
Please add the string, `icon`. 
`$material-icon-font-name:` & `$material-icon-font-url`

Reply:
Done